distributed/tests/test_steal.py

```diff
@@ -1350,3 +1372,316 @@ def test_steal_worker_state(ws_with_running_task):
     assert "x" not in ws.tasks
     assert "x" not in ws.data
     assert ws.available_resources == {"R": 1}
+
+
+@pytest.mark.parametrize("recompute_saturation", [True, False])
```

Comment:
you can also define this once as a fixture instead of parametrizing everything

```python
@pytest.fixture(params=[True, False])
def recompute_saturation(request):
    yield request.param

```

Reply:
Good point, I'll do some cleanup on the added tests now.